views/js/runner/provider/qti.js

```diff
@@ -166,12 +168,15 @@ define([
                         }
 
                         self._item = null;
+                        return;
```

Comment:
I don't think this is required as it is already the last line of the function :)